v2/emailpassword/custom-ui/handling-session-tokens.mdx

```diff
@@ -200,12 +200,20 @@ fileprivate class NetworkManager {
 We highly recommend using our frontend SDK to handle session token management. It will save you a lot of time.
 :::
 
```

Comment:
Under each of the sections (For Web, For React-native, For Android, For iOS), add a note about if cookie or header (authorization bearer) based method is used.